requirements.txt

```diff
@@ -6,6 +6,8 @@ dnspython
 passlib
 requests>=2.9.1
 zeroc-ice
+slugify
+#requests_cache
```

Comment:
Oh, I see. It was left commented out when I pulled last (a while ago...) so I thought I better let it be. But sorry for creating that peculiar problem in the first place...